staging/src/k8s.io/apimachinery/pkg/types/uid.go

```diff
@@ -20,3 +20,9 @@ package types
 // don't ONLY use UUIDs, this is an alias to string.  Being a type captures
 // intent and helps make sure that UIDs and names do not get conflated.
 type UID string
+
+// A pod UID which has been translated/resolved to the representation known to kubelets.
+type ResolvedPodUID UID
```

Comment:
Specifically `pkg/kubelet/types`, or a more specific one if these are not related.